WikiWatershed/model-my-watershed
# Add Mid-Atlantic ARA layer

## Overview

Add painted ARA layer to Coverage Tab. Exact names and descriptions subject to change. There also isn't a legend, but since there's only one value, do we need it?

Connects #2886 

### Demo

![2018-07-19 16 27 30](https://user-images.githubusercontent.com/1430060/42968352-ff1555be-8b70-11e8-9d74-9a8a8b85f9d3.gif)

## Testing Instructions

* Check out this branch
* Go to [:8000/](http://localhost:8000/) and select the "Mid-Atlantic ARA" layer in Coverage tab
* Ensure it is available, works as expected, and has an opacity slider
* Go to [:8000/?bigcz](http://localhost:8000/?bigcz) and ensure the layer is not available
